docs/questions/native-editor/sql-parameters.md

```diff
@@ -324,16 +221,123 @@ WHERE
 {% endraw %}
 ```
 
-### SQL syntax
+## Customizing dropdown lists and search box values
```

Comment:
@abdhabli This is the only section you really need to review, as the other stuff is just moving around.